# HG changeset patch
# User Rik <rik@octave.org>
# Date 1379616360 25200
#      Thu Sep 19 11:46:00 2013 -0700
# Node ID 0aa77acf22f5140af7387711ef429d7a148fd1b0
# Parent  4e86bb1901d6e048d9f58c98b2f70fc0f8a8c98f
doc: Add seealso references between cla, clf, reset functions.

* libinterp/corefcn/graphics.cc(Freset): Add @seealso links to cla, clf.

* scripts/plot/cla.m, scripts/plot/clf.m: Add @seealso link to reset.

diff --git a/libinterp/corefcn/graphics.cc b/libinterp/corefcn/graphics.cc
--- a/libinterp/corefcn/graphics.cc
+++ b/libinterp/corefcn/graphics.cc
@@ -8515,16 +8515,17 @@ Undocumented internal function.\n\
 
 DEFUN (reset, args, ,
   "-*- texinfo -*-\n\
 @deftypefn {Built-in Function} {} reset (@var{h}, @var{property})\n\
 Remove any defaults set for the handle @var{h}.  The default figure\n\
 properties of @qcode{\"position\"}, @qcode{\"units\"},\n\
 @qcode{\"windowstyle\"} and @qcode{\"paperunits\"} and the default axes\n\
 properties of @qcode{\"position\"} and @qcode{\"units\"} are not reset.\n\
+@seealso{cla, clf}\n\
 @end deftypefn")
 {
   int nargin = args.length ();
 
   if (nargin != 1)
     print_usage ();
   else
     {
diff --git a/scripts/plot/cla.m b/scripts/plot/cla.m
--- a/scripts/plot/cla.m
+++ b/scripts/plot/cla.m
@@ -28,17 +28,17 @@
 ##
 ## If the optional argument @qcode{"reset"} is specified, delete all child
 ## objects including those with hidden handles and reset all axis properties
 ## to their defaults.  However, the following properties are not reset:
 ## Position, Units.
 ##
 ## If the first argument @var{hax} is an axes handle, then operate on
 ## this axis rather than the current axes returned by @code{gca}.
-## @seealso{clf}
+## @seealso{clf, delete, reset}
 ## @end deftypefn
 
 ## Author: Ben Abbott <bpabbott@mac.com>
 ## Created: 2008-10-03
 
 function cla (varargin)
 
   if (nargin > 2)
diff --git a/scripts/plot/clf.m b/scripts/plot/clf.m
--- a/scripts/plot/clf.m
+++ b/scripts/plot/clf.m
@@ -32,17 +32,17 @@
 ## properties to their defaults.  However, the following properties are not
 ## reset: Position, Units, PaperPosition, PaperUnits.
 ##
 ## If the first argument @var{hfig} is a figure handle, then operate on
 ## this figure rather than the current figure returned by @code{gcf}.
 ## 
 ## The optional return value @var{h} is the graphics handle of the figure
 ## window that was cleared.
-## @seealso{cla, close, delete}
+## @seealso{cla, close, delete, reset}
 ## @end deftypefn
 
 ## Author: jwe
 
 function h = clf (varargin)
 
   if (nargin > 2)
     print_usage ();
